components/Banner.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="banner">
+    <transition name="slide">
+      <div class="banner-container" v-if="open">
+        <p>{{ message }}</p>
+        <nacelle-button
+          class="nacelle-banner-button"
+          @click="goTo(buttonPath)"
+          color="secondary"
+          >{{ buttonText }}</nacelle-button
+        >
```

Comment:
@levimichael This may be another point of standardization to set for the team.

The Vue docs uses the same syntax as above when putting components in the template, but I've been using the syntax `<NacelleButton>` in mine. To me, it makes it easier to see when a custom component is being used.

I don't have a preference, just something I noticed between our codebases.

Reply:
@drewclem good call out -- I think for now, since we're in this in-between area where we are patching up some functionality, we don't need to be as perscriptive about it as I will want us to be once the docs and website are merged together.